sonarqube/content.md

```diff
@@ -23,217 +23,39 @@ ulimit -u 4096
-	    -v $SONARQUBE_HOME/extensions:/opt/sonarqube/extensions \
-	    -v $SONARQUBE_HOME/logs:/opt/sonarqube/logs \
-	    -v $SONARQUBE_HOME/data:/opt/sonarqube/data \
-	    %%IMAGE%%:community-beta
-	```
+## Upgrading 
 
-## Upgrade SonarQube
-
-### From 7.9.x LTS to another 7.9.x LTS version
-
-Take a look at the [Upgrade Guide](https://docs.sonarqube.org/latest/setup/upgrading/).  
-No specific Docker operations are needed, just use the new tag.
-
-### From 7.9.x LTS to 8.x
-
-Follow the next section instructions, but instead of upgrading from folders to folders, you will migrate your volumes to new folders.
-
-> Starting with the version 8.0, you can move your `sonar.jdbc.username` and `sonar.jdbc.password` value to `sonar.properties` instead of passing them with the `-e` flag.
-
-### From 8.x
-
-Follow these steps to upgrade SonarQube:
-
-1.	Create a new `sonarqube_home_new` folder and backup your old folder.
-
-2.	Update the environment variable `$SONARQUBE_HOME` pointing to it:
-
-	```console
-	$ mkdir /path/to/your/filesystem/sonarqube_home_new
-	$ export SONARQUBE_HOME=/path/to/your/filesystem/sonarqube_home_new
-	```
-
-3.	Initialize the new `SONARQUBE_HOME` with `--init`:
-
-	```console
-	$ docker run --rm \
-	    -v $SONARQUBE_HOME/conf:/opt/sonarqube/conf \
-	    -v $SONARQUBE_HOME/extensions:/opt/sonarqube/extensions \
-	    -v $SONARQUBE_HOME/data:/opt/sonarqube/data \
-	    %%IMAGE%%:community-beta --init  
-	```
-
-4.	Take a look at the [Upgrade Guide](https://docs.sonarqube.org/latest/setup/upgrading/) for information on:
-
-	-	Manually installing the non-default plugins
-	-	Updating the contents of sonar.properties and wrapper.conf
-	-	Copying the Oracle DB JDBC driver if needed  
-		  
-
-5.	Stop and remove the sonarqube container (a restart is not enough as the environment variables are only evaluated during the first run, not during a restart):
-
-	```console
-	$ docker stop %%IMAGE%%
-	$ docker rm %%IMAGE%%
-	```
-
-6.	Run docker:
-
-	```console
-	$ docker run -d --name sq -p 9000:9000 \
-	    -v $SONARQUBE_HOME/conf:/opt/sonarqube/conf \
-	    -v $SONARQUBE_HOME/extensions:/opt/sonarqube/extensions \
-	    -v $SONARQUBE_HOME/logs:/opt/sonarqube/logs \
-	    -v $SONARQUBE_HOME/data:/opt/sonarqube/data \
-	    %%IMAGE%%:community-beta
-	```
-
-7.	Browse to `http://yourSonarQubeServerURL/setup` and follow the setup instructions.
-
-8.	Reanalyze your projects to get fresh data.
+For upgrade instructions, see Upgrading from the Docker Image on the [Upgrade the Server](https://docs.sonarqube.org/latest/setup/upgrading/) page.
 
 ## Advanced configuration
 
-#### Use parameters via Docker environment variables
-
-The database can be configured with the following SonarQube properties used as environment variables: `sonar.jdbc.username`, `sonar.jdbc.password` and `sonar.jdbc.url`.
-
-```console
-$ docker run -d --name sonarqube \
-    -p 9000:9000 \
-    -e sonar.jdbc.username=sonar \
```

Comment:
We should not recommend `sonar.jdbc....` environment variables ever TBH, it was always weird. It's even an invalid format for env variables- you can't create an env var with dots under linux or macos (`export sonar.jdbc.username="abc"` will throw an error). My opinion is we should recommend `SONARQUBE_JDBC_USERNAME` style env vars for old images and `SONAR_JDBC_USERNAME` style for new images.